src/data/chart.json

```diff
@@ -4,290 +4,1242 @@
+          "blockStat": {
+            "type": "number",
+            "subtitle": "Elections",
+            "description": "Votes",
+            "statistic": {
+              "aggregate": "sum",
+              "unique": false
+            }
+          }
+        },
+        {
+          "title": "Registered voters vs total votes cast",
+          "visuals": {
+            "type": "pie",
+            "table": "allRegisteredAccreditedVoters",
+            "x": "voters",
+            "y": "total"
+          },
+          "blockStat": {
+            "type": "number",
+            "subtitle": "Elections",
+            "description": "Registered Voters",
+            "statistic": {
+              "aggregate": "sum",
+              "unique": false
+            }
+          }
+        },
+        {
+          "title": "Registered voters vs total votes cast",
+          "visuals": {
+            "type": "pie",
+            "table": "allVotesPerCandidates",
+            "x": "candidate",
+            "y": "total"
+          },
+          "blockStat": {
+            "type": "number",
+            "subtitle": "Elections",
+            "description": "Cast Votes",
+            "statistic": {
+              "aggregate": "sum",
+              "unique": false
+            }
+          }
+        },
+        {
+          "title": "Proportion of seats held by women in national parliaments (%)",
+          "visuals": {
+            "type": "column",
+            "table": "allWomenInParliaments",
+            "x": "womenParliamentYear",
+            "y": "total"
+          },
+          "blockStat": {
+            "type": "number",
+            "subtitle": "Women In Parliament",
+            "description": "Seat Proportion",
+            "statistic": {
+              "unique": true,
+              "unit": "%"
+            }
+          }
+        },
+        {
+          "title": "Position held by women in government",
+          "visuals": {
+            "type": "column",
+            "table": "allWomenInGovernments",
+            "x": "position",
+            "y": "total"
+          },
+          "blockStat": {
+            "type": "number",
+            "subtitle": "Women in Government",
+            "description": "Total Count",
+            "statistic": {
+              "aggregate": "sum"
+            }
+          }
         }
       ]
   },
   {
       "sectionTitle": "Health",
       "charts": [{
           "title": "People using at least basic drinking water services (% of population)",
-          "visuals": [{
+          "visuals": {
             "type": "column",
             "table": "allAccessToBasicServices",
-            "reference": {},
             "x": "accessToBasicServicesYear",
             "y": "total"
-          }]
+          },
+          "blockStat": {
+            "type": "number",
+            "subtitle": "Health",
+            "description": "Basic Dring Water Services",
+            "statistic": {
+              "unique": true,
```

Comment:
just curious, what does unique mean ?

Reply:
@karimkawambwa in aggregation, we current aggregate sum of unique x, say I have number of doctors `female` and `male` , it will sum all female and sum all male.

 I added unique, so when it is false, it will give me sum of all doctors.
In Number visual we use either of the two, and more